### DISCUSSION TITLE
The problem with the iPad

### DISCUSSION THREAD

**Discussion by MyNameTony1000** [KingSora/OverlayScrollbars#507] [Q&A] [answered]
Hello.
I recently found that the page and all scripts stop working on the iPad (Safari version 12.1.2). Not only libraries, but also those that I wrote myself. By turning off and connecting all the plugins in turn - it turned out that this is due to the connection of this plugin. Without it, there is no problem with the page and everything works as it should. Initialization on the body of the page.
This is an initialization code:
`
this.data.pageScroll = OverlayScrollbars(
        {
          target: document.body,
        },
        {
          overflow: { x: 'hidden' },
          showNativeOverlaidScrollbars: true,
          scrollbars: {
            visibility: 'auto',
            autoHide: 'scroll',
            autoHideDelay: 300,
          },
        }
      );
`
Maybe someone knows what is the problem?)

**Comment by KingSora** [comment, marked as answer]
Good day @MyNameTony1000 :)

I would like to investigate in that issue.. I would need a minimal reproducable example on jsfiddle, codesandbox or stackblitz, so I can be sure the issue is happening in a neutral environment

**Reply by MyNameTony1000** [reply]
Hello, thanks for responding so quickly. After a long study, it turned out that the problem is on my side, the plugin works fine)
Sorry for distracting

**Reply by MyNameTony1000** [reply]
Is it possible to remove this discussion?)

**Reply by KingSora** [reply]
@MyNameTony1000 I can't remove it. But you can mark it as answered so its at least not marked as "open".

**Reply by MyNameTony1000** [reply]
thanks again)
